Cython/Utility/Coroutine.c

```diff
@@ -2123,11 +2123,11 @@ static int __Pyx_patch_abc(void) {
     static int abc_patched = 0;
     if (CYTHON_REGISTER_ABCS && !abc_patched) {
         PyObject *module;
-        module = PyImport_ImportModule((PY_MAJOR_VERSION >= 3) ? "collections.abc" : "collections");
+        module = PyImport_ImportModule((CYTHON_CONDITION(PY_MAJOR_VERSION >= 3) ? "collections.abc" : "collections"));
```

Comment:
This might be the cleaner solution
```cpp
#if PY_MAJOR_VERSION >= 3
    module = PyImport_ImportModule("collections.abc");
#else
    module = PyImport_ImportModule("collections");
#endif
```